Python/pylifecycle.c

```diff
@@ -1240,6 +1240,7 @@ init_interp_main(PyThreadState *tstate)
         if (_Py_get_xoption(&config->xoptions, L"uops") != NULL) {
             enabled = 1;
         }
+        enabled = 1;
```

Comment:
I think that this is a hard-coded value for your experimentation, It should be removed right?
```suggestion
```

Reply:
Yes, thanks.